### PR TITLE
kernel: bump 6.1 to 6.1.73

### PR DESCRIPTION
**Pull request by graysky2**
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.73

Patches automatically rebased.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod

### CODE REVIEW

**Comment by antnyfls** [comment]
The patch https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch require some manual intervention, because this changes https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.72&id=8b160f2fba777a27b912bcd2488e95d21f6936f0

**Comment by graysky2** [comment]
I am traveling and might have have a chance to look at this. If you have a fix please share it.

**Comment by graysky2** [comment]
I am traveling and won't have a chance to look at this. If anyone has some time to spin up a fix, please, share it.

[@nbd168](https://github.com/nbd168) - since you authored [650-netfilter-add-xt_FLOWOFFLOAD-target.patch](https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch) can you take a look?

**Comment by graysky2** [comment]
[@nbd168](https://github.com/nbd168) - since you authored [650-netfilter-add-xt_FLOWOFFLOAD-target.patch](https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch) can you take a look at my rebase?

**Review by DragonBluep — APPROVED** [review]

File: target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch [hunk 1]
```diff
@@ -654,7 +654,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +};
 +
 +static int nf_flow_rule_route_inet(struct net *net,
-+				   const struct flow_offload *flow,
++				   struct flow_offload *flow,
```
[the remark this answers is not in the document]
Comment (graysky2) [reply target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch:657]: I am confused.  The current patch does not have the `const` on that line.  What modification are you requesting?

[the remark this answers is not in the document]
Comment (DragonBluep) [reply target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch:657]: No, I approved your changes.
image

File: target/linux/mediatek/patches-6.1/962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch
Comment (dick902) [inline target/linux/mediatek/patches-6.1/962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch:1]: This need to remove as already committed in  https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e

File: target/linux/mediatek/patches-6.1/961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch
Comment (dick902) [inline target/linux/mediatek/patches-6.1/961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch:1]: This need to remove as already committed in  https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e

File: target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch (hunk 1 above)
[the remark this answers is not in the document]
Comment (graysky2) [reply target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch:657]: OK, thanks :+1:

File: target/linux/mediatek/patches-6.1/962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch
Comment (graysky2) [reply target/linux/mediatek/patches-6.1/962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch:1]: Done, thanks

File: target/linux/mediatek/patches-6.1/961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch
Comment (graysky2) [reply target/linux/mediatek/patches-6.1/961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch:1]: Done, thanks

**Review by dick902 — APPROVED** [review]

**Comment by DragonBluep** [comment]
> This need to remove as already committed in [e1a6c72](https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e)

I don't understand why we have to remove these two patches. They were introduced just two days ago... https://github.com/openwrt/openwrt/commit/ede34465de2176229590028fba622d54841ab40c

**Comment by dick902** [comment]
> > This need to remove as already committed in [e1a6c72](https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e)
> 
> I don't understand why we have to remove these two patches. They were introduced just two days ago... [ede3446](https://github.com/openwrt/openwrt/commit/ede34465de2176229590028fba622d54841ab40c)

Those patch was duplicated as already merged in to master yesterday!  https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e

**Comment by Gingernut1978** [comment]
This PR, after merge, removes both these patches which seems incorrect:

961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch
962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch

**Comment by graysky2** [comment]
I am also confused by me removing them as I think merging this PR would delete them from master, no?

**Comment by Gingernut1978** [comment]
LGTM

**Comment by dick902** [comment]
That's work for me! THX!

**Pull request merged**